lib/streamlit/runtime/uploaded_file_manager.py

```diff
@@ -39,6 +39,16 @@ class UploadFileUrlInfo(NamedTuple):
     delete_url: str
 
 
+class DeletedFile:
+    """Return this from st.file_uploader and st.camera_input deserialize (so they can
```

Comment:
nit: maybe the first line of this docstring can be something like

"Represents a deleted file in deserialized values for `st.file_uploader` and `st.camera_input`"

Then the current docstring can be the description underneath this summary.